plugins/net_plugin/net_plugin.cpp

```diff
@@ -4597,7 +4597,6 @@ namespace eosio {
    // called from any thread
    void connections_manager::start_conn_timers() {
       start_conn_timer(connector_period, {}, timer_type::check); // this locks mutex
-      start_conn_timer(connector_period, {}, timer_type::stats); // this locks mutex
```

Comment:
I wonder if this was mistakenly left in during some sort of refactor? `connection_statistics_monitor()` is clearly only valid when `update_p2p_connection_metrics` is a valid function (it even `assert()`s on it). I thought maybe `connection_statistics_monitor()` drives some other non-prometheus statistics when prometheus isn't enabled, but it doesn't seem so.